sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -565,16 +680,31 @@ class depends_on {
   public:
     template<typename... NodeTN>
     depends_on(NodeTN... nodes);
+
+    template<typename... EventTN>
+    depends_on(EventTN... events);
 };
 }
 ----
 
 The API for explicitly adding nodes to a `command_graph` includes a
 `property_list` parameter. This extension defines the `depends_on` property to
-be passed here. `depends_on` defines any `node` objects for the created node to
-be dependent on, and therefore form an edge with. These nodes are in addition to
-the dependent nodes identified from the command-group requisites of the created
-node.
+be passed here. `depends_on` may be used in two ways:
+
+* Passing nodes from the same `command_graph` which will create dependencies and
```

Comment:
"Passing nodes from another command_graph" is not covered.

Reply:
This is mentioned in the next sentence I believe (emphasis added):

> For dynamic events, **graph-limited events from a different graph** or <<event-terminology,
regular SYCL events>>, an external dependency is created between this node and
the command that is associated with the event.

Reply:
I think the next sentence talks about 'event', not 'node'.

Reply:
Ah you are correct, I misread it. I've added a bullet point above which explicitly talks about nodes from other graphs.